modules/packages/SortedSet.chpl

```diff
@@ -40,13 +40,13 @@ module SortedSet {
   private use Treap;
   private use Reflection;
   private use IO;
-  public use Sort only DefaultComparator;
+  public use Sort only defaultComparator;
 
   // TODO: remove this module and its public use when the deprecations have been
   // removed
   pragma "ignore deprecated use"
   private module HideDeprecatedReexport {
-    public use Sort only defaultComparator;
+    public use Sort only DefaultComparator;
   }
```

Comment:
can't this whole thing be deleted?

Reply:
After reading the rest of the PR, I believe this is because otherwise `DefaultComparator` will disappear without a deprecation warning if a users has `use SortedSet`. If that is correct, please ignore this comment

Reply:
Yeah, that's right :)